src/fidesops/service/connectors/saas_connector.py

```diff
@@ -41,22 +41,31 @@ def __init__(self, configuration: ConnectionConfig):
         self.endpoints = self.saas_config.top_level_endpoint_dict  # type: ignore
         self.collection_name: Optional[str] = None
 
-    def query_config(self, node: TraversalNode) -> SaaSQueryConfig:
+    def query_config(
+        self, node: TraversalNode, privacy_request: PrivacyRequest
+    ) -> SaaSQueryConfig:
         """
         Returns the query config for a given node which includes the endpoints
         and connector param values for the current collection.
         """
         # store collection_name for logging purposes
         self.collection_name = node.address.collection
         return SaaSQueryConfig(
-            node, self.endpoints, self.secrets, self.saas_config.data_protection_request  # type: ignore
+            node,
+            self.endpoints,
+            self.secrets,
+            privacy_request,
+            self.saas_config.data_protection_request,
         )
 
     def test_connection(self) -> Optional[ConnectionTestStatus]:
         """Generates and executes a test connection based on the SaaS config"""
         test_request: SaaSRequest = self.saas_config.test_request  # type: ignore
-        prepared_request: SaaSRequestParams = SaaSRequestParams(
-            method=test_request.method, path=test_request.path
+        prepared_request = map_param_values(
```

Comment:
Previously we were only mapping `method` and `path` but the `test_request` for this connector requires other params to be mapped. This was just changed to leverage an existing util (`map_param_values`) which maps all parameters.